en/toolbar/toolbar.md

```diff
@@ -1,6 +1,7 @@
 # Main Toolbar
 
```

Comment:
This is good. Tempting to add an image of the whole toolbar here giving the icons context